api/src/main/java/net/kyori/adventure/text/Component.java

```diff
@@ -1709,6 +1710,38 @@ default void detectCycle(final @NotNull Component that) {
     return this.append(builder.build());
   }
 
+  /**
+   * Wraps the provided component in an empty component with the given style.
+   *
+   * <p>This method can be used to set the "default" style for a component, whilst still allowing the component to override the style.</p>
+   *
+   * @param like the component which will be wrapped
+   * @param style style to use by default
+   * @return wrapped component
+   * @since 4.10.0
+   */
+  @Contract(pure = true)
+  static @NotNull Component wrap(final @NotNull ComponentLike like, final @NotNull Style style) {
```

Comment:
The componentlike parameter should just be called `component` instead of like

Reply:
was just copying the others ^^